pytorch_lightning/trainer/trainer.py

```diff
@@ -172,7 +172,7 @@ def __init__(
         prepare_data_per_node: Optional[bool] = None,
         plugins: Optional[Union[PLUGIN_INPUT, List[PLUGIN_INPUT]]] = None,
         amp_backend: str = "native",
```

Comment:
Should amp backend be optional too? 

It doesn't match the defaults with precision=32 on the trainer.

On the other hand, if this were optional, users would need to set this flag and precision=16 for mixed precision training

Reply:
I think it is fine this way. Let's confusing to the user who knows native is the default one.